src/client/dfs/SConscript

```diff
@@ -7,6 +7,41 @@ def scons():
 
     denv = env.Clone()
 
+    """If Lustre installed use its headers instead of our local defines"""
+    try:
+        import os
+        with open('/etc/os-release') as f:
+            for line in f:
+                if 'ID' in line:
+                    line = line.strip()
+                    id, distro = line.strip().split('=')
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'id'
(pylint-invalid-name) Invalid variable name "id"
(pylint-unused-variable) Unused variable 'id'